src/main/java/heronarts/glx/ui/component/UISwitch.java

```diff
@@ -52,6 +52,7 @@ public UISwitch() {
 
   public UISwitch(float x, float y) {
     super(x, y, WIDTH, SWITCH_SIZE);
+    this.keyEditable = true;
```

Comment:
Did this to allow up/down keys on UISwitch but can't tell if it will have unintended consequences.